File: app/src/main/java/ie/dublinbuspal/android/data/remote/rss/DublinBusRssApi.java
package ie.dublinbuspal.android.data.remote.rss;

import ie.dublinbuspal.android.data.remote.rss.xml.Rss;

import retrofit2.Call;
import retrofit2.http.GET;

public interface DublinBusRssApi {

    String BASE_URL = "http://www.dublinbus.ie/";

    @GET("RSS/Rss-news/")
    Call<Rss> getRssNews();

}

File: service/src/main/kotlin/ie/dublinbuspal/service/DublinBusRssApi.kt
package ie.dublinbuspal.service

import ie.dublinbuspal.service.model.rss.RssResponseXml
import io.reactivex.Single
import retrofit2.http.GET

interface DublinBusRssApi {

    @GET("RSS/Rss-news/")
    fun getRssNews(): Single<RssResponseXml>

}
